### PR TITLE
adds note of how to deal with int64 in js to InteractingWithCode page…

### PR DESCRIPTION
**Pull request by WizardOfArc**
… - may have to be elaborated further

### CODE REVIEW

**Comment by WizardOfArc** [comment]
so weird - I only changed a documentation file

**Comment by kripken** [comment]
I think it would be better to link to the FAQ entry, so the info isn't duplicated in both places.

**Comment by WizardOfArc** [comment]
agreed...  I'll change that entry to a link.

**Comment by WizardOfArc** [comment]
I changed it to a link (or at least I _think_ I did) . I haven't used Sphinx / rst before.

**Comment by kripken** [comment]
To test it, you can enter `site/` and run `make html`. Then you can find the HTML files under `build/html`.

**Comment by kripken** [comment]
No problem, I tested it locally. This diff makes it work:
```
diff --git a/site/source/docs/getting_started/FAQ.rst b/site/source/docs/getting_started/FAQ.rst
index 1bb2b95..fab387e 100644
--- a/site/source/docs/getting_started/FAQ.rst
+++ b/site/source/docs/getting_started/FAQ.rst
@@ -473,7 +473,8 @@ To check if this is the issue you are seeing, build with ``-s PRECISE_F32=1``. T
 
 (This is not an issue for wasm, which has native float types.)
 
-: _Int64-in-JS:
+.. _Int64-in-JS:
+
 How do I pass int64_t and uint64_t values from js into wasm functions?
 ======================================================================
 
diff --git a/site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst b/site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst
index 0d9eea7..e153a5f 100644
--- a/site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst
+++ b/site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst
@@ -749,7 +749,8 @@ for defining the binding:
 .. _Box2D: https://github.com/kripken/box2d.js/#box2djs
 .. _Bullet: https://github.com/kripken/ammo.js/#ammojs
 
-Dealing with the fact that Javascript does not represent int64 types.
-=====================================================================
-See :ref:`FAQ <Int64-in-JS>`
+Dealing with the fact that Javascript does not represent int64 types
+====================================================================
+
+See :ref:`the FAQ <Int64-in-JS>`.
 ```
Please add a commit with that here.

**Comment by curiousdannii** [comment]
Personally it would make more sense to me to have the proper documentation in the interacting-with-code page, and have the FAQ link to that.

**Comment by WizardOfArc** [comment]
I think @curiousdannii might be right...  it would make more sense to put the longer description in `interacting-with-code` and just the link in `FAQ`

File: site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst [hunk 1]
```diff
@@ -748,3 +748,9 @@ for defining the binding:
 .. _tests/test_core.py: https://github.com/kripken/emscripten/blob/1.29.12/tests/test_core.py#L4597
 .. _Box2D: https://github.com/kripken/box2d.js/#box2djs
 .. _Bullet: https://github.com/kripken/ammo.js/#ammojs
+
+.. _Int64-in-JS:
+Dealing with the fact that Javascript does not represent int64 types.
```
Comment (kripken) [inline site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst]: there needs to be a space after the `..` line (as in the patch i pasted)

**Comment by kripken** [comment]
I'm fine with it either way, just as long as we don't have the text in both places.

**Comment by curiousdannii** [comment]
This would be a bigger change than what's being proposed here, but it would be great to have full examples showing:

 - for calling a func with a 64 bit int showing both the C function declaration, and how to call it with ccall
 - for a function return a 64bit int showing how JS can access getTempRet0 and combine it with the return val

**Pull request closed without merging**